fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/scalar/MvNormalizePredicateRule.java

```diff
@@ -70,7 +70,7 @@ public class MvNormalizePredicateRule extends NormalizePredicateRule {
     @Override
     public ScalarOperator visitCompoundPredicate(CompoundPredicateOperator predicate,
                                                  ScalarOperatorRewriteContext context) {
-        Map<String, ScalarOperator> sorted = Maps.newTreeMap(String.CASE_INSENSITIVE_ORDER);
+        Map<String, ScalarOperator> sorted = Maps.newTreeMap();
```

Comment:
if changed, the column name would be case-sensitive, which means behavior change ?

Reply:
add more comments, PTAL.